EDDiscovery/PanelAndPopOuts.cs

```diff
@@ -60,6 +60,7 @@ public enum PanelIDs        // id's.. used in tab controls, and in button pop ou
             Trilateration,          // 28
             Settings,               // 29
             ScanGrid,               // 30
+            Compass,
```

Comment:
Need the "// 31" for debug. ;)

Reply:
The whole point of an enum is that you don't need to know the numbers...

Reply:
Yes, I understand what yo mean, and I agree with you. However, is something was request by @robbyxp1 when I included my first panel, ScanGrid. So, I think is something's needed somewhat, or possibly only a requirement...

Reply:
Actually.. the reason i put the numbers in is because this is saved as an integer in the DB to mark the panel, hence i needed to know the number of the enums ;-)